src/sentry/api/endpoints/organization_repository_details.py

```diff
@@ -20,6 +20,7 @@ class RepositorySerializer(serializers.Serializer):
             # XXX(dcramer): these are aliased, and we prefer 'active' over 'visible'
             ("visible", "visible"),
             ("active", "active"),
+            ("hidden", "hidden"),
```

Comment:
Just curious, why do we set choices as a tuple with duplicate values?

Reply:
i think sometimes you can have different values of what you allow to be sent to the API and what the serializer outputs as the value https://github.com/getsentry/sentry/blob/master/src/sentry/api/validators/servicehook.py#L10